doc/developer/animation.md

```diff
@@ -1,50 +1,79 @@
-\page develanimation API: Character animation
-[TOC]
+# Animation in Radium
 
 The Radium Engine provides a set of classes dedicated to animation,
-which can be found in Ra::Core::Animation.
+which can be found in `Core/Animation`.
 
 ## Animation Basics
 
-An Ra::Core::Animation::Animation is a set of key poses, each associated to a key time.
-When looking for the pose at a specific animation time,
-the 2 closest surrounding key poses are linearly interpolated to build the querried pose.
+In order to animate a digital character, one may choose from two main animation techniques:
+ - skeleton-based animation, in which the character's mesh is bound to an
+animation skeleton composed of joints, sometimes called bones.
+ - cage-based animation, in which the character's mesh is bound to a set of
+points defining an emglobing deformation cage.
 
-Though the Ra::Core::Animation::Animation class only provides keyframing for poses, which are sets of transformation matrices,
-the Ra::Core::Animation::KeyFrame class provides a more generic interface, usable for any type of animated data.
+In both cases, the character's mesh is bound to the handles (skeleton joints or
+cage points) through a set of skinning weights.
+During the animation, the handles transformations are first updated with respect
+to the animation pose, then the mesh is deformed by combining the handles
+transformations through the skinnning weights.
 
-## Character Animation
+## Character Animation in Radium
 
-The Radium Engine also provides the basic classes for character animation.
-Starting from a Ra::Core::Animation::Handle, which represents a deformation metaphor, such as Ra::Core::Animation::Cage or Ra::Core::Animation::Skeleton,
-one can associate influence weights to each vertex of a mesh within a `WeightMatrix` (rows representing vertices, columns handles).
+The Radium Engine provides the basic classes for character animation.
+
+Starting from a `HandleSystem`, which represents a deformation metaphor, such as `Cage` or `Skeleton`,
```

Comment:
When you compile the documentation, makes sure that the classes are well recognized by doxygen.
Usually you need to add the namespace, e.g. `Ra::Core::HandleSystem` should work.

Reply:
I can I compile the Doc?